openetl_utils/scd_utils.py

```
@@ -0,0 +1,652 @@
+    try:
+        full_table = f"{schema}.{table_name}" if schema else table_name
+        df = (
+            spark_session.read.format("jdbc")
+            .option("url", con_string)
+            .option("dbtable", full_table)
+            .option("driver", driver)
+            .load()
+        )
+        if df.head(1):
+            return df
+        return None
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)